app/docs/dev/policies/meshaccesslog.md

```diff
@@ -0,0 +1,293 @@
+---
+title: MeshAccessLog (beta)
+---
+
+With the MeshAccessLog policy you can easily set up access logs on every data plane in a mesh.
+
+{% warning %}
+This policy uses new policy matching algorithm and is in beta state,
+it should not be mixed with [TrafficLog](/docs/{{ page.version }}/policies/trafficlog).
+{% endwarning %}
+
+{% tip %}
+In the rest of this page we assume you have already configured your observability tools to work with Kuma.
+If you haven't already read the [observability docs](/docs/{{ page.version }}/explore/observability).
+{% endtip %}
+
+## TargetRef support matrix
+
+| TargetRef type    | top level | to  | from |
+|-------------------|-----------|-----|------|
+| Mesh              | ✅         | ✅   | ✅    |
+| MeshSubset        | ✅         | ❌   | ❌    |
+| MeshService       | ✅         | ✅   | ❌    |
+| MeshServiceSubset | ✅         | ❌   | ❌    |
+| MeshGatewayRoute  | ✅         | ❌   | ❌    |
+| MeshHTTPRoute     | ❌         | ❌   | ❌    |
```

Comment:
```suggestion
```

These aren't actually supported/don't exist

Reply:
I was basing them on validator code so we should remove that from validation as well. https://github.com/kumahq/kuma/blob/b6fdbda848230bb26925f76c94fb6e7248403b43/pkg/plugins/policies/meshaccesslog/api/v1alpha1/validator.go#L32